apps/ui/src/views/Auction/Bids.vue

```diff
@@ -1 +1,122 @@
-<template>Bids</template>
+<script setup lang="ts">
+import { useInfiniteQuery } from '@tanstack/vue-query';
+import { AuctionNetworkId, formatPrice, getOrders } from '@/helpers/auction';
+import { AuctionDetailFragment } from '@/helpers/auction/gql/graphql';
+import { _c, _t, shortenAddress } from '@/helpers/utils';
+
+const LIMIT = 20;
+
+const props = defineProps<{
+  network: AuctionNetworkId;
+  auctionId: string;
+  auction: AuctionDetailFragment;
+}>();
+
+const {
+  data,
+  fetchNextPage,
+  hasNextPage,
+  isPending,
+  isFetchingNextPage,
+  isError
+} = useInfiniteQuery({
+  initialPageParam: 0,
+  queryKey: ['auction', props.network, props.auctionId, 'orders'],
+  queryFn: ({ pageParam }) =>
+    getOrders(props.auctionId, props.network, {
+      first: LIMIT,
+      skip: pageParam
+    }),
+  getNextPageParam: (lastPage, pages) => {
+    if (lastPage.length < LIMIT) return null;
+
+    return pages.length * LIMIT;
+  }
+});
+
+async function handleEndReached() {
+  if (!hasNextPage.value) return;
+
+  fetchNextPage();
+}
+</script>
+
+<template>
+  <div>
+    <UiSectionHeader label="Bids" sticky />
+    <UiColumnHeader class="z-40 overflow-hidden gap-3">
+      <div class="max-w-[218px] w-[218px] truncate">Bidder</div>
+      <div class="grow w-[40%] truncate">Amount</div>
+      <div class="flex max-w-[144px] w-[144px] items-center truncate">Date</div>
```

Comment:
```suggestion
      <div class="max-w-[144px] w-[144px] truncate">Date</div>
```

Reply:
The flex class is not needed anymore since we only have one inner element (the flex was used when there was a icon, for the sorting order)